openpype/plugins/publish/extract_review.py

```diff
@@ -1014,14 +1014,10 @@ def audio_args(self, instance, temp_data, duration_seconds):
             return audio_in_args, audio_filters, audio_out_args
 
         for audio in audio_inputs:
-            # NOTE modified, always was expected "frameStartFtrack" which is
-            # STRANGE?!!! There should be different key, right?
-            # TODO use different frame start!
             offset_seconds = 0
-            frame_start_ftrack = instance.data.get("frameStartFtrack")
-            if frame_start_ftrack is not None:
-                offset_frames = frame_start_ftrack - audio["offset"]
-                offset_seconds = offset_frames / temp_data["fps"]
+            frame_start = instance.data.get("frameStart")
```

Comment:
Are you sure changing the default behaviour is OK?

Reply:
As explained in the comment :
```python
# NOTE modified, always was expected "frameStartFtrack" which is
# STRANGE?!!! There should be different key, right?
# TODO use different frame start!
```
without frame_start_ftrack we never can't apply offset params correctly.